Hardware/Hardware/CardReader/CardReaderConnectionOptions.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+public struct CardReaderConnectionOptions {
+    public let builtInOptions: BuiltInCardReaderConnectionOptions?
+
+    public init(builtInOptions: BuiltInCardReaderConnectionOptions?) {
+        self.builtInOptions = builtInOptions
+    }
+}
+
+public struct BuiltInCardReaderConnectionOptions {
+    public let termsOfServiceAcceptancePermitted: Bool
+
+    public init(termsOfServiceAcceptancePermitted: Bool) {
+        self.termsOfServiceAcceptancePermitted = termsOfServiceAcceptancePermitted
+    }
```

Comment:
Should we split these structs in different files? No strong preference, just commenting for consistency, as seems to be how we set the different typealiases for models in Yosemite (I haven't checked them all, so could be just coincidence on the ones I opened 👍 )

Reply:
I think they're so tightly coupled and small that it makes more sense to keep them together, but perhaps you're right in  the sense that it'll help preserve history if/when they grow for more options. I'll do that before I merge